aws/resource_aws_fsx_backup.go

```diff
@@ -0,0 +1,174 @@
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+
+		CustomizeDiff: customdiff.Sequence(
+			SetTagsDiff,
+		),
+	}
+}
+
+func resourceAwsFsxBackupCreate(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).fsxconn
+	defaultTagsConfig := meta.(*AWSClient).DefaultTagsConfig
+	tags := defaultTagsConfig.MergeTags(keyvaluetags.New(d.Get("tags").(map[string]interface{})))
+
+	input := &fsx.CreateBackupInput{
+		ClientRequestToken: aws.String(resource.UniqueId()),
+		FileSystemId:       aws.String(d.Get("file_system_id").(string)),
+	}
+
+	if len(tags) > 0 {
+		input.Tags = tags.IgnoreAws().FsxTags()
+	}
+
+	result, err := conn.CreateBackup(input)
+	if err != nil {
+		return fmt.Errorf("Error creating FSx Backup: %w", err)
+	}
+
+	d.SetId(aws.StringValue(result.Backup.BackupId))
+
+	log.Println("[DEBUG] Waiting for FSx backup to become available")
+	if _, err := waiter.BackupAvailable(conn, d.Id()); err != nil {
+		return fmt.Errorf("error waiting for FSx backup (%s) to be available: %w", d.Id(), err)
+	}
+
+	return resourceAwsFsxBackupRead(d, meta)
+}
+
+func resourceAwsFsxBackupUpdate(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).fsxconn
+
+	if d.HasChange("tags_all") {
+		o, n := d.GetChange("tags_all")
+
+		if err := keyvaluetags.FsxUpdateTags(conn, d.Get("arn").(string), o, n); err != nil {
+			return fmt.Errorf("error updating FSx Lustre File System (%s) tags: %w", d.Get("arn").(string), err)
+		}
+	}
+
+	return resourceAwsFsxBackupRead(d, meta)
+}
+
+func resourceAwsFsxBackupRead(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).fsxconn
+	defaultTagsConfig := meta.(*AWSClient).DefaultTagsConfig
+	ignoreTagsConfig := meta.(*AWSClient).IgnoreTagsConfig
+
+	backup, err := finder.BackupByID(conn, d.Id())
+	if !d.IsNewResource() && tfresource.NotFound(err) {
+		log.Printf("[WARN] FSx Backup (%s) not found, removing from state", d.Id())
+		d.SetId("")
+		return nil
+	}
+
+	if err != nil {
+		return fmt.Errorf("error reading FSx Backup (%s): %w", d.Id(), err)
+	}
+
+	d.Set("arn", backup.ResourceARN)
+	d.Set("type", backup.Type)
+
+	fs := backup.FileSystem
+	d.Set("file_system_id", fs.FileSystemId)
+
+	if backup.KmsKeyId != nil {
+		d.Set("kms_key_id", backup.KmsKeyId)
+	}
+
+	d.Set("owner_id", backup.OwnerId)
+
+	tags := keyvaluetags.FsxKeyValueTags(backup.Tags).IgnoreAws().IgnoreConfig(ignoreTagsConfig)
+
+	//lintignore:AWSR002
+	if err := d.Set("tags", tags.RemoveDefaultConfig(defaultTagsConfig).Map()); err != nil {
+		return fmt.Errorf("error setting tags: %w", err)
+	}
+
+	if err := d.Set("tags_all", tags.Map()); err != nil {
+		return fmt.Errorf("error setting tags_all: %w", err)
+	}
+
+	return nil
+}
+
+func resourceAwsFsxBackupDelete(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).fsxconn
+
+	request := &fsx.DeleteBackupInput{
+		BackupId: aws.String(d.Id()),
+	}
+
+	_, err := conn.DeleteBackup(request)
```

Comment:
Extra log line helps when running sweepers.

```suggestion
	log.Printf("[INFO] Deleting FSx Backup: %s", d.Id())
	_, err := conn.DeleteBackup(request)
```